contracts/base/FallbackManager.sol

```diff
@@ -28,14 +29,9 @@ contract FallbackManager is SelfAuthorized {
         internalSetFallbackHandler(handler);
     }
 
-    function ()
+    fallback()
         external
-        payable
```

Comment:
By removing this we revert if you send eth to the safe with random data. (Which should be ok ... but maybe we should keep it for that) .... If we keep the `payable` we can remove the `receive` function to minimize the code